src/unix/lwt_unix.ml

```diff
@@ -659,6 +659,114 @@ let write_string ch buf pos len =
   let buf = Bytes.unsafe_of_string buf in
   write ch buf pos len
 
+module IO_vectors =
+struct
+  type _bigarray =
+    (char, Bigarray.int8_unsigned_elt, Bigarray.c_layout) Bigarray.Array1.t
```

Comment:
Could you use `Lwt_bytes.t` instead of a new type definition here?  It is the [same definition](https://github.com/ocsigen/lwt/blob/master/src/unix/lwt_bytes.mli#L25)